teams/migration/membership.json

```diff
@@ -14,6 +14,7 @@
         "3pointer",
         "5kbpers",
         "CharlesCheung96",
+        "charleszheng44",
```

Comment:
Items are in alphabetical order, where `charleszheng44 ` starts with a lower case letter.

Reply:
Good catch!